tools/mfassignr/mfassignr_histnoise.xml

```diff
@@ -0,0 +1,44 @@
+<tool id="mfassignr_histnoise" name="MFAssignR HistNoise" version="@TOOL_VERSION@+galaxy0">
+    <description>Noise level assessment using the HistNoise.</description>
```

Comment:
```suggestion
    <description>Noise level assessment using the HistNoise</description>
```

Reply:
resolved in 2dbdd36efd69b7f996b258139927160bca954948